src/include/daos/mem.h

```diff
@@ -516,6 +536,10 @@ umem_id_equal(struct umem_instance *umm, umem_id_t ummid_1, umem_id_t ummid_2)
 #define umem_id_equal_typed(umm, tmmid_1, tmmid_2)			\
 	umem_id_equal(umm, (tmmid_1).oid, (tmmid_2).oid)
 
+#define umem_id_equal_off(umm, off_1, off_2)				\
+	umem_id_equal(umm, umem_off2id(umm, off_1), umem_off2id(umm, off_2))
```

Comment:
I guess we probably never compare two values with invalid bits set.   This would actually treat them as equal (both NULL).   I should probably rename this

Reply:
I removed it in latest patch